hc/accounts/tests/test_check_token.py

```diff
@@ -8,34 +8,35 @@ def setUp(self):
         super(CheckTokenTestCase, self).setUp()
         self.profile.token = make_password("secret-token")
         self.profile.save()
-        self.client.login(username="alice", password="password")
 
     def test_it_shows_form(self):
-        r = self.client.get("/accounts/check_token/alice/secret-token/")
-        self.assertContains(r, "You are about to log in")
+        """ Test response for login"""
+        response = self.client.get("/accounts/check_token/alice/secret-token/")
+        self.assertContains(response, "You are about to log in")
 
     def test_it_redirects(self):
-        r = self.client.post("/accounts/check_token/alice/secret-token/")
-        self.assertRedirects(r, "/checks/")
+        """ Test redirection correct login and token"""
+
+        response = self.client.post(
+            "/accounts/check_token/alice/secret-token/")
+        self.assertRedirects(response, "/checks/")
 
         # After login, token should be blank
         self.profile.refresh_from_db()
         self.assertEqual(self.profile.token, "")
 
-    # Login and test it redirects already logged in
     def test_login_and_redirects_already_logged_in(self):
+        """ Login and test it redirects already logged in"""
         payload = {"email":"alice@example.org", "password":"password"}
-        response = self.client.post("/accounts/login/", payload, content = "application/json")
-        # chcek redirection to chceks
+        response = self.client.post(
+            "/accounts/login/", payload, content="application/json")
+        # check redirection to chceks
         self.assertRedirects(response, "/checks/")
 
-    # Login with a bad token and check that it redirects
     def test_login_with_bad_token(self):
-        payload = {"email":"alice@example.org", "password":"password"}
-        # bad token
-        self.profile.token = 'dcdf'
+        """ Login with a bad token and check that it redirects """
+
+        # test using random token
         response = self.client.post("/accounts/check_token/alice/rgrn/")
```

Comment:
Check the [docs](https://docs.djangoproject.com/en/1.10/ref/urlresolvers/#reverse) for `reverse` usage